locale/all.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
How is this file created? Now it is done locally I assume? This would need to be automated. Have you looked at how this is setup for qcs-ui-common?

Reply:
run locale:generate to generate all.json

Reply:
But it is not included in the pipeline. Every time the files are changed that command should run in some way. We actually only need to create the combined file when we are building the package

Reply:
okay, I gonna add 'yarn run locale:generate &&' into 'build' sciprt

Reply:
fixed